EIPS/eip-5164.md

```diff
@@ -0,0 +1,211 @@
+---
+eip: 5164
+title: Cross-Chain Execution
+description: This specification defines an interface that supports execution across EVM networks.
```

Comment:
```suggestion
description: Defines an interface that supports execution across EVM networks.
```

Reply:
Fixed in this commit: https://github.com/ethereum/EIPs/pull/5164/commits/da5558d00a3b5ab768df5b7d7336d92bd1f1c085#diff-41efe0617f861cc7330b025ecc86d55fe777d3c503208d83dca04258de4f5f41R4